meinberlin/react/contrib/Card.jsx

```diff
@@ -9,12 +9,32 @@ const translated = {
 }
 
 export const Card = (props) => {
-  const { item, permissions, children } = props
+  const { item, idx, permissions, children } = props
+
+  // needed as BO JS library does not work with async libraries
+  const [hover, setHover] = useState()
+
+  const handleClick = () => {
+    document.getElementById('card-link-' + idx).click()
+  }
+
+  const handleMouseEnter = () => {
+    setHover(!hover)
+  }
+
+  const handleMouseLeave = () => {
+    setHover(!hover)
+  }
 
   return (
     <article className="modul-card" data-add-clickable-area="smart">
```

Comment:
no when there is a link on the card, only the link and title should be clickable, copy from their styleguide: 'To avoid duplicate links, you can also make the title clickable with the data attribute data-add-clickable-area="smart" if there is already a detail link in the section with the class card__body'

I kept the data attribute thinking they might implement async compatible js but actually you are right, it's not looking like it's anytime soon so will remove